phoenicis-javafx/src/main/java/org/phoenicis/javafx/JavaFXApplication.java

```diff
@@ -43,6 +44,7 @@ public static void main(String[] args) {
     @Override
     public void start(Stage primaryStage) {
         primaryStage.getIcons().add(new Image(getClass().getResourceAsStream("views/common/phoenicis.png")));
+        primaryStage.initStyle(StageStyle.DECORATED);
```

Comment:
This is the default according to the API: https://docs.oracle.com/javase/8/javafx/api/javafx/stage/Stage.html#initStyle-javafx.stage.StageStyle-

Reply:
I know but it's the only thing I could imagine to be wrong. Or do you have any idea what else could explain the behavior described in the issue?

Reply:
I wouldn't eliminate a misconfiguration of the user somewhere. I think we should let the issue stay open and add a `help wanted` tag. Maybe someone else has a similar issue. This would allow us to better isolate the issue.

Reply:
In that case we are blocked for alpha 3.

Reply:
I don't see a reason why this needs to be contained in Alpha 3? I think it is fine to move this to a later alpha version especially if we are unsure of the cause and can't reproduce the issue and therefore test a bugfix.

Reply:
Currently it's marked for alpha 3. I'm fine doing it later.

Reply:
Then let us move it to a later release